h/static/scripts/annotator/guest.coffee

```diff
@@ -68,7 +70,6 @@ module.exports = class Guest extends Annotator
             this.publish(event, args)
       formatter: (annotation) =>
         formatted = {}
-        formatted.uri = @getHref()
```

Comment:
Although, actually -- does the currently client use this field at all? Could removing it cause problems where annotations made with the new client don't work in older deployed versions?